components/library/uthash/Makefile

```diff
@@ -0,0 +1,34 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 Benjamin S. Osenbach
+#
+
+BUILD_BITS=32_and_64 # for binaries or 32_and_64 for libraries
+BUILD_STYLE=justmake
```

Comment:
BUILD_BITS=NO_ARCH should be enough (or 32 or 64) because you'll only copy the header files and they are identical for both variants.